bedrock/settings/static_media.py

```diff
@@ -1072,6 +1072,18 @@
         ),
         'output_filename': 'css/thunderbird-start-bundle.css',
     },
+    'firefox-retention-thank-you-a': {
+        'source_filenames': (
+            'css/etc/firefox/retention/thank-you-a.scss',
+        ),
+        'output_filename': 'css/firefox-retention-thank-you-a-bundle.css',
+    },
+    'firefox-retention-thank-you-b': {
+        'source_filenames': (
+            'css/etc/firefox/retention/thank-you-b.scss',
+        ),
+        'output_filename': 'css/firefox-retention-thank-you-b-bundle.css',
+    }
```

Comment:
Can we prefix these bundle names with `etc-`? I think that'll be a good pattern going forward to avoid confusion with bundles used on `/firefox/` URLs.